specs/albums.spec.js

```diff
@@ -0,0 +1,124 @@
+"use strict";
+
+const chakram = require("chakram");
+const expect = chakram.expect;
+const api = require("./utils/api");
+const data = require("../server/data.json");
+
+describe("ALBUMS", () => {
+  describe("CREATE", () => {
+    let addedId;
+    it("should add an album", () => {
+      return chakram
+        .post(api.url("albums"), {
+          userId: 1,
```

Comment:
Could you please create a variable for it? You could use the object values to check response bode.
e.g.: expect(album).to.have.json("data.title", `${variable.title}`);

Reply:
added templete literals